core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java

```diff
@@ -917,6 +917,15 @@ public void testLoadTable() {
         .containsAll(properties.entrySet());
   }
 
+  @Test
+  public void loadTableWithNonExistingNamespace() {
```

Comment:
To be consistent, better name it `testLoadTableWithNonExistingNamespace`

Reply:
good point, updated